social_core/tests/backends/open_id_connect.py

```diff
@@ -53,6 +53,9 @@ class OpenIdConnectTest(
     issuer: str  # id_token issuer
     openid_config_body: str
     key: dict[str, str]
+    allow_invalid_at_hash = False
+    skip_invalid_at_hash = False
+    access_token_kwargs: dict[str, Any]
```

Comment:
typing here was required to ensure that `pyright` stops throwing errors on `prepare_access_token_body` for `tamper_message` not being a string.